styles/main.css

```diff
@@ -221,14 +221,18 @@ div.favourites input:checked + label + div
   display:flex;
   flex-wrap: nowrap;
 }
-label h3
+div.favourites label h3
 {
   font-family: 'Times New Roman', Times, serif;
   font-style: italic;
   font-size: 20px;
   color: rgba(255, 165, 0);
   background:  #a7272a;
   border: 3px solid black;
+  opacity: 50%;
+}
+div.favourites input:checked + label h3 {
+  opacity: 100%;
```

Comment:
How about a little visual differentiation between the selected tab and the unselected tabs?